src/Security/Member.php

```diff
@@ -382,7 +382,7 @@ public function canLogIn()
      *
      * @return bool
      */
-    public function isLockedOut()
+    protected function isLockedOut()
```

Comment:
It can't be now- if this represents what it says then it may be useful as a public method